platform/ios/CHANGELOG.md

```diff
@@ -69,6 +69,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Points of interest have clearer, localized VoiceOver hints in styles that use version 8 of the Mapbox Streets source. ([#13525](https://github.com/mapbox/mapbox-gl-native/pull/13525))
 * Added `MGLLoggingConfiguration` and `MGLLoggingBlockHandler` that handle error and fault events produced by the SDK. ([#13235](https://github.com/mapbox/mapbox-gl-native/pull/13235))
 * Fixed random crashes during app termination. ([#13367](https://github.com/mapbox/mapbox-gl-native/pull/13367))
+* Specify custom NSURLSessionDelegate with Info.plist key MGLNSURLSessionDelegateClassName. ([#13491](https://github.com/mapbox/mapbox-gl-native/pull/13491))
```

Comment:
When using verbs at the beginning of the changelog entry use the past tense.
Enclose any class/method/keyword with ``.
This entry should be in the latest version: 4.9.0